proto/message_contents/signature.proto

```diff
@@ -8,12 +8,21 @@ option go_package = "github.com/xmtp/proto/go/message_contents";
 // Signature represents a generalized public key signature,
 // defined as a union to support cryptographic algorithm agility.
 message Signature {
-    // The format for an ECDSA signature bytes and the recovery bit
+    // ECDSA signature bytes and the recovery bit
     message ECDSACompact {
         bytes bytes = 1; // compact representation [ R || S ], 64 bytes
         uint32 recovery = 2; // recovery bit
     }
+    // ECDSA signature bytes and the recovery bit
+    // produced by xmtp-js::PublicKey.signWithWallet function, i.e.
+    // EIP-191 signature of a "Create Identity" message with the key embedded.
+    // Used to sign identity keys.
+    message WalletECDSACompact {
```

Comment:
We should start using this for identity key signatures. Maybe we can do that for the new SignedPublicKeys only to make the logic easier to handle.

Reply:
Sounds good to me. We are already going to have to refactor the `ContactBundle` for V2